lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlockBlobFileSystem.cs

```diff
@@ -216,5 +221,10 @@ public FileStatus GetFileStatus(Uri remoteFileUri)
 
             return new FileStatus(lastModifiedTime.Value.DateTime, blobReference.Properties.Length);
         }
+
+        private string GetUriPrefix()
+        {
+            return _client.BaseUri.AbsoluteUri.TrimEnd('/');
```

Comment:
removed

Reply:
Im concerned that we are changing the behavior of Uri because that is the expected behavior. You can probably just return _client.BaseUri as "file" would get removed when you append a new path (i.e. new Uri(_client.BaseUri, relativePath) will result in "http://www.fake.com/_relativePath_"

Reply:
The GetUriPrefix() helper function was removed.